common/apps.go

```diff
@@ -0,0 +1,85 @@
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/types"
+	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func runApps() {
+	Context("default apps", func() {
+		var customFormatterKey format.CustomFormatterKey
+		BeforeEach(func() {
+			customFormatterKey = format.RegisterCustomFormatter(func(value interface{}) (string, bool) {
```

Comment:
Needed to have a better output when test fails. This is what it looks like

![image](https://github.com/giantswarm/cluster-test-suites/assets/627038/98eba32d-4afe-49ba-9a94-c8e3df1279f0)